tomb-common/src/metadata.rs

```diff
@@ -100,16 +100,9 @@ impl FsMetadata {
 
         // TODO: Can we get away with merging these somehow?
         // Put the forests in the store
-        let metadata_forest_cid =
-            store_forest(&self.metadata_forest, metadata_store, metadata_store).await?;
-        let _metadata_forest_cid =
-            store_forest(&self.metadata_forest, metadata_store, content_store).await?;
-        let content_forest_cid =
-            store_forest(&self.content_forest, content_store, metadata_store).await?;
-        let _content_forest_cid =
-            store_forest(&self.content_forest, content_store, content_store).await?;
-        assert_eq!(metadata_forest_cid, _metadata_forest_cid);
-        assert_eq!(content_forest_cid, _content_forest_cid);
+        let forest_cid = store_forest(&self.forest, metadata_store, metadata_store).await?;
+        let _forest_cid = store_forest(&self.forest, metadata_store, content_store).await?;
```

Comment:
Underscored variables should never be used/referenced (this is getting used in the assert)